BAZProject/TrendingMoviesModule/TrendingMoviesViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TrendingMoviesViewModel.swift
+//  BAZProject
+//
+//  Created by gescarcega on 29/03/23.
+//
+
+import UIKit
+
+class TrendingMoviesViewModel {
+    
+    var movies: [Movie]?
+    
+    func getMovies(completion: @escaping () -> Void ) {
+        TrendingAPIService().getMovies { [weak self] movies in
```

Comment:
Este metodo rompe SRP (Single Responsibility Principle), su unica responsabilidad en teoria solo es obtener las peliculas, pero le damos una adicional al buscar de donde obtener las peliculas instanciando `TrendingAPIService`, podríamos inyectar este como dependencia en su lugar en el ViewModel y así solo reemplazar como
```suggestion
        loader.getMovies { [weak self] movies in
```

Reply:
Duda: Si un módulo requiriera más de un servicio, como se podría manejar? Se hizo de tal modo que si al guardar un favorito se requiera algún servicio distinto, el ViewModel fuera capaz de preparar y mandar al servicio indicado. Si de hace como comentas, no quedaría totalmente casado sólo con un remoteHandler (loader)?